ando/tests/test_AnDOChecker.py

```diff
@@ -91,6 +91,10 @@ def test_nonAuthorizedDataFilesError(self):
         path = Path(dir_path) / "dataset" / "exp-nonAuthorizedDataFilesError"
         self.assertEqual(CHK.is_valid(path)[0], False)
 
+    def test_missingDataFile(self):
```

Comment:
Maybe it would also be good to check if the expected number of error is correct.